docroot/themes/contrib/civictheme/civictheme_library/components/00-base/mixins/_icon.scss

```diff
@@ -48,6 +48,14 @@
   @error '"#{$dimension}" is not an available dimension for size #{$size} at breakpoint #{$breakpoint}.';
 }
 
+//
+// Compile all icon symbols into a single map.
+//
+// stylelint-disable-next-line scss/at-function-pattern
+@function _civictheme-icon-symbols-all() {
+  @return list.join($civictheme-icon-symbols-default, $civictheme-icon-symbols);
```

Comment:
please use the same system of merging of symbols maps as colours

Reply:
Updated to follow map.merge